third_party/path.h

```diff
@@ -350,6 +350,13 @@ inline void copy_file(const filesystem::path &from, const filesystem::path &to)
     ofs << ifs.rdbuf();
 }
 
+inline filesystem::path
+current_path()
+{
+    char cwd[PATH_MAX];
+    return getcwd(cwd, sizeof(cwd));
```

Comment:
Pretty sure this is POSIX only

Reply:
It does work on Windows (though it's technically deprecated): https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/getcwd?view=msvc-170